R/env.R

```diff
@@ -723,14 +723,26 @@ env_has <- function(env = caller_env(), nms, inherit = FALSE) {
   map_lgl(nms, exists, envir = get_env(env), inherits = inherit)
 }
 
-#' Get an object from an environment
+#' Get or set an object in an environment
 #'
-#' `env_get()` extracts an object from an enviroment `env`. By
-#' default, it does not look in the parent environments.
+#' @description
+#'
+#' * `env_get()` extracts an object from an enviroment `env`. By
+#'   default, it does not look in the parent environments.
+#'
+#' * `env_set()` will assign or reassign a binding in `env` if
```

Comment:
The default value for `create` is now a function of `inherit`, cf last commit. It is not clear what's the best behaviour is (`env_set()` vs `<<-`), but otoh it is clear that the main purpose of the inheriting case is to override an existing binding so an error is the safest option by default.